src/settings.py

```diff
@@ -33,6 +33,7 @@ class GlobalSettings:
     pretty_print_when_loading = False
     # Used for random seeding in test environment.
     test_env_seed_offset = 10000
+    make_interaction_videos = False
```

Comment:
while we're here... two additional tiny changes would be nice...
* take out the `make_failure_videos` line from args.py and put it here instead: `make_failure_videos = False`
* for the `make_videos` in args.py, rename it to `make_test_videos` (but keep it in args.py). you'll have to update make_videos -> make_test_videos in two places in main.py as well

Reply:
copying from slack: these should all live in `args.py` instead, and we should still make the second change